.github/workflows/test.yml

```diff
@@ -0,0 +1,57 @@
+---
+name: Testing
```

Comment:
⛏️ Maybe "dotnet" or ".Net" testing -- since the database tests are specific, this should be as well.

Reply:
I kinda thought of this as the location for all logic-level and therefore "generic" tests, whereas the other file is specific infrastructure testing for a database.